samples/brokers/http/Dockerfiles/standalone

```diff
@@ -0,0 +1,51 @@
+ARG PLATFORM=amd64
+ARG CROSS_BUILD_TARGET=x86_64-unknown-linux-gnu
+ARG BROKER=http
+
+FROM ${PLATFORM}/rust:1.47 as build
+
+ARG BROKER
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    g++ \
+    ca-certificates \
+    curl \
+    libssl-dev \
+    pkg-config
+
+RUN rustup component add rustfmt --toolchain 1.47.0-x86_64-unknown-linux-gnu
+
+WORKDIR /${BROKER}
+
+RUN echo '[workspace]' > ./Cargo.toml && \
+    echo 'members = ["shared", "samples/brokers/http"]' >> ./Cargo.toml
+
+COPY ./samples/brokers/${BROKER} ./samples/brokers/${BROKER}
+COPY ./shared ./shared
```

Comment:
i'm not sure that shared is needed for the http broker, is it?

Reply:
Fixed: https://github.com/deislabs/akri/pull/135/commits/05d00db60e1d514be9a36a240033dc9466486134